sparklemotion/src/main/java/com/ifttt/sparklemotion/Decor.java

```diff
@@ -50,8 +50,8 @@
      */
     final boolean withLayer;
 
-    private Decor(@NonNull View contentView, @NonNull Page page, boolean layoutBehind, Animation slideInAnimation,
-            Animation slideOutAnimation, boolean withLayer) {
+    Decor(@NonNull View contentView, @NonNull Page page, boolean layoutBehind, Animation slideInAnimation,
```

Comment:
The constructor is meant to be private so that it cannot be accessed from outside, the only way to construct a new Decor is through the builder.